Ginger/GingerCoreNET/ActionsLib/ActionsConversion/ActionConversionUtils.cs

```diff
@@ -27,7 +27,8 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Reflection;
-using System.Text;
+using System.Threading.Tasks;
+using static Amdocs.Ginger.CoreNET.BusinessFlowToConvert;
```

Comment:
you need to set eConversionStatus enum outside the BusinessFlowToConvert class (on same .cs file but not under same class) so then you won't need this static using

Reply:
Done